build/Dockerfile.multiarch

```diff
@@ -0,0 +1,10 @@
+FROM golang:buster AS builder
+
+COPY . /build
+WORKDIR /build
+RUN CGO_ENABLED=0 go build -o ./_output/bin/piraeus-operator --ldflags '-extldflags "-static"' -gcflags all=-trimpath=. --asmflags all=-trimpath=. github.com/piraeusdatastore/piraeus-operator/cmd/manager
+
+FROM gcr.io/distroless/static-debian10:latest
```

Comment:
Please switch to `gcr.io/distroless/static:latest` and add a `USER 1001` line to keep the old default uid.